interview-questions.md

```diff
@@ -8,44 +8,49 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would super be used in a Ruby class?
 
-Your answer:
+Your answer: to create a sub class
 
-Researched answer:
+Researched answer: Super is a keyword used to call a method of the same name from the parnet class of the current class.
 
 2. What is a gem?
 
-Your answer:
+Your answer: A piece code that can be called upon easily but is not hard coded in the main piece of code source.
 
-Researched answer:
+Researched answer: A package piece of a library that can easily be included in a Ruby project.
 
-3. What is a relational database? Are there other kinds of databases?
+3. What is a relational database? Are there other kinds of databases? 
 
-Your answer:
+Your answer: A database that is used to organize and store data using tables, columns, and rows.
 
-Researched answer:
+Researched answer: Data stored in a collction of tables where each table represents something different and the relationships between each table are defined with keys. NoSql, Graph, columnar, time-series, and in memory data base.
 
 4. What are primary keys? Why purpose does a primary key serve?
 
-Your answer:
+Your answer: The base key for all tables
 
-Researched answer:
+Researched answer: Each table has a primary key, and the primary key is a unique identifier for each row. Makes sure each row can be identified for other tables. 
 
 5. What are the five HTTP verbs? What is each verb's corresponding CRUD action?
 
-Your answer:
+Your answer: GET, Post, put, patch and delete
 
 Researched answer:
+Get: Read action in CRUD
+Post: Create action in CRUD
+PUT: Update action in CRUD, used to update or replace the data of a resource
+PATCH: Update action in CRUD, used to update specific fields of a resource
+Delete: Delete action of CRUD
```

Comment:
Your answers are mostly accurate (see above).  I really encourage you to expand your answers so they fully demonstrate your knowledge, ie. use examples, your own experience, etc.  Nice work overall!